packages/apidom-ls/src/services/validation/linter-functions.ts

```diff
@@ -1009,9 +1011,35 @@ export const standardLinterfunctions: FunctionItem[] = [
     functionName: 'apilintOpenAPIPathTemplateValid',
     function: (element: Element) => {
       if (isStringElement(element)) {
-        return true;
+        const pathItemElement = (element.parent as MemberElement).value as ObjectElement;
+
+        if (pathItemElement.length === 0) {
+          return true;
+        }
+
+        const pathTemplateResolveParams: { [key: string]: string } = {};
+
+        pathItemElement.forEach((operation) => {
+          const parameters = (operation as OperationElement).get('parameters');
+          if (isArrayElement(parameters)) {
+            parameters.forEach((parameter) => {
+              if (isParameterElement(parameter)) {
+                const allowedLocation = ['path', 'query'];
+                if (allowedLocation.includes(toValue(parameter.in))) {
+                  pathTemplateResolveParams[toValue(parameter.name) as string] = 'placeholder';
+                }
+              }
+            });
+          }
+        });
+
```

Comment:
Code is not collecting `ParamterElements` for `PathItemElement` at all, as I've described in "semantic code template":

```ts
        // we're treating PathItemElement polymorphically as ObjectElement as this function has to support all OpenAPI versions
        const pathItemElement = (element.parent as MemberElement).value as ObjectElement;
        if (isArrayElement(pathItemElement.get('parameters'))) {
          parameterElements.push([...pathItemElement.get('parameters')]);
        }
```